examples/pipelines/controlnet-interior-design/README.md

```diff
@@ -0,0 +1,60 @@
+# ControlNet Interior Design Pipeline
+
+## Introduction
+This example demonstrates an end-to-end fondant pipeline to collect and process data for the training of a [ControlNet](https://github.com/lllyasviel/ControlNet) model, focusing on images related to interior design.
+
+## Pipeline Overview
+
+The image below shows the entire pipeline and its workflow.
+
+![Image](./assets/controlnet-interior-design-pipeline.png)
```

Comment:
If we want to keep the image, should we instead create a global assets folder at the root of the repo?

Reply:
We can add an `art` folder under the `docs` folder.